### PR TITLE
CI: add version compatible test using hypothesis

### PR DESCRIPTION
**Pull request by zhoucheng361**

### CODE REVIEW

File: .github/scripts/testVersionCompatible.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+        print('start status')
+        output = self.exec_check_output([juicefs, 'status', self.meta_url])
+        uuid = json.loads(output.decode('utf8').replace("'", '"'))['Setting']['UUID']
+        assert len(uuid) != 0
+        if self.mounted:
+            sessions = json.loads(output.decode('utf8').replace("'", '"'))['Sessions']
+            assert len(sessions) != 0 
+        print('status succeed')
+
+    @rule(juicefs=st.sampled_from(JFS_BIN))
```
Comment (davies) [inline .github/scripts/testVersionCompatible.py:186]: We only provide backwards compatibility, so old version may not mount volume formatted by new version.

**Review by davies — CHANGES_REQUESTED** [review]
only check backward

File: .github/scripts/testVersionCompatible.py (hunk 1 above)
Comment (zhoucheng361) [reply .github/scripts/testVersionCompatible.py:190]: ok, will check the formatted version when mount.

Comment (zhoucheng361) [reply .github/scripts/testVersionCompatible.py]: have add backwards check using assume.
image

**Review by sanwan — APPROVED** [review]

**Pull request merged**